packages/zowe-explorer/src/configuration/Profiles.ts

```diff
@@ -530,15 +539,15 @@ export class Profiles extends ProfilesCache {
             switch (choice) {
                 case "project":
                     for (const file of existingLayers) {
-                        if (file.user) {
+                        if (!file.global) {
                             await this.openConfigFile(file.path);
                         }
                     }
                     Gui.showMessage(this.manualEditMsg);
                     break;
                 case "global":
                     for (const file of existingLayers) {
-                        if (file.global) {
+                        if (file.path.includes(FileManagement.getZoweDir())) {
```

Comment:
I think the change on this line could be reverted to address Fernando's comment 😋 
```suggestion
                        if (file.global) {
```